hive-metastore/src/main/java/org/apache/iceberg/hive/HiveClientPool.java

```diff
@@ -73,7 +73,8 @@ protected IMetaStoreClient newClient() {
     } catch (MetaException e) {
       throw new RuntimeMetaException(e, "Failed to connect to Hive Metastore");
     } catch (Throwable t) {
-      if (t.getMessage().contains("Another instance of Derby may have already booted")) {
+      if (t.getMessage() != null
```

Comment:
Having trouble to add tests for this. Since `GET_CLIENT` is static. 
@nastra Any pointers will help.

Reply:
try to look at the existing tests in `TestHiveClientPool` and see how things could be mocked so that we get the right exception